### PR TITLE
Updated image-width shortcode to use named parameters.

### PR DESCRIPTION
**Pull request by micronAMZN**
Ordered parameters are still supported for backwards compatibility.

Signed-off-by: Mike Cronin <58789750+micronAMZN@users.noreply.github.com>

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Review by FiniteStateGit — APPROVED** [review]

File: content/docs/contributing/to-docs/style-guide/media.md [hunk 1]
```diff
@@ -26,21 +26,21 @@ Image output:
...
 The `image-width` shortcode adds an image with alternate text and restricts the image's width. The `image-width` shortcode can ensure image sizes are consistent within a topic, and that large images don't scale overly large in wide browser windows. Use this method when adding `.svg` diagrams, very large images, and in topics with multiple images where consistent image widths are desirable.
 
-`image-width` takes three double-quoted parameters in order:
+`image-width` takes three double-quoted named parameters:
 
-1. image link
-1. width
-1. alternate text
+1. `src="/images/<image.png>"` - Image file path.
+1. `width="<200>"` - Scale the image by specifying a width in pixels.
```
Comment (willihay) [inline content/docs/contributing/to-docs/style-guide/media.md:32]: I think you want either a placeholder description: `width="<image width>"` or an actual example: `width="200"`, but not the mixture of an actual example inside of angle brackets. I'm fine with either, but slightly prefer the placeholder description, for consistency with the placeholders used for the other parameters.
```suggestion
1. `width="<image width>"` - Scale the image by specifying a width in pixels.
```

File: content/smoketest.md [hunk 2]
```diff
@@ -452,21 +452,21 @@ You can add inline O3DE GUI icons with the `icon` shortcode. Icon `.svg` files a
...
 The `image-width` shortcode adds an image with alternate text and restricts the image's width. The `image-width` shortcode can ensure image sizes are consistent within a topic, and that large images and `.svg` diagrams don't scale overly large in wide browser windows.
 
-`image-width` takes three double-quoted parameters in order:
+`image-width` takes three double-quoted named parameters:
 
-1. image link
-1. width
-1. alt text
+1. `src="/images/<image.png>"` - Image file path.
+1. `width="<200>"` - Scale the image by specifying a width in pixels.
```
Comment (willihay) [inline content/smoketest.md:458]: See previous comment.
```suggestion
1. `width="<image width>"` - Scale the image by specifying a width in pixels.
```

**Review by willihay — CHANGES_REQUESTED** [review]

**Comment by willihay** [comment]
@micronAMZN Heads up, this PR now has a conflict to resolve before it can be merged.

**Comment by micronAMZN** [comment]
I'll submit a new PR after 1566 gets merged so to minimize dealing with merge conflicts.

**Pull request closed without merging**